### ISSUE TITLE
Is this project active?

### ISSUE THREAD

**Issue by agb80** [collective/collective.hostout#44]
We are looking for a tool to use in combination with our buildout recipes to deploy our python apps. Is this project active, are there anybody using it? Any links that I would review about user experiences?

**Comment by djay** [comment]
Yes its active. We just added capability to help deploy via docker. We use
this tool both with and without docker for all our deployments. You should
use the master branch as there hasn't been a release in awhile.

**Comment by agb80** [comment]
Thanks a lot for your answer @djay, do you have any links that I can refer in order to get some insides on how you use it?

Thanks in advance again!

**Comment by djay** [comment]
What are you trying to do? What is the problem you are encountering?

**Comment by agb80** [comment]
Honestly, I do't have done any test jet, I'm on a research phase trying to evaluate available tools and reduce to a short list of two or three for testing.

This is how I think my deployments must look like:

1. Schedule deployment as we need to defer it to a time where no one is using the service, midnight for example.
2. Stop the service. Supervisor installed on buildout folder.
3. Make a backup of databases.
4. Update buildout recipe from remote repository
5. Run buildout to reflect changes done on recipe.
6. Clone each of databases and apply schema upgrades
7. Drop old databases and rename new ones to old names
8. If anything goes wrong between 5 and 7 get back to original state and send a mail with error.
9. Restart service

Could you please assist me to know if that is possible to accomplish with houstout? I'm specially interested on the rollback feature as would like to use an existing task instead of create my own.
